_sass/vrsite/staff.scss

```diff
@@ -0,0 +1,62 @@
+.staff {
+	display: flex;
+	justify-content: space-around;
+	flex-wrap: wrap;
+
+	.card {
+		display: flex;
+		flex-direction: column;
+		flex: 0 1 200px;
+		justify-content: flex-start;
+		margin: 1em;
+		padding: 1em;
+		border-style: solid;
+		border-width: 1px;
+		height: auto;
+
+		&:nth-child(odd):last-child {
+			display: none;
+		}
+		
+		&:nth-child(even):last-child {
+			visibility: hidden;
+			
+			@include media-query($on-palm) {
+				display: none;
+			}
+		}
+
+		.thumb {
+			display: block;
+			max-height: 240px;
+			max-width: 100%;
+			margin: auto;
+		}
+	
+		.text-wrapper {
+
+			.title {
+				text-align: center;
+			}
+
+			.desc {
+				overflow: hidden;
+				text-overflow: ellipsis;
+				display: -webkit-box;
+				-webkit-box-orient: vertical;
+				-webkit-line-clamp: 4; /* number of lines to show */
+				line-height: 1.5em;        /* fallback */
+				height: 6em;       /* fallback */
+			}
+
+			.links {
+				float: right;
+				height: 1em;
+	
+				.proj-link-icon > .svg-icon {
+					fill: $dark-background-color;
+				}
+			}
+		}	
+	}
+}
```

Comment:
Let's aim to separate out the gallery as its own feature and apply it here and on the projects page. (#93) That way, we can minimize the amount of duplicated CSS. For now, can we move the duplicate CSS as an include? Any variations between the two implementations can be referenced as variables.